node/src/test/scala/coop/rchain/node/configuration/commandline/Ed25519PubKeyConverterSpec.scala

```diff
@@ -13,9 +13,9 @@ class Ed25519PubKeyConverterSpec extends PropSpec with GeneratorDrivenPropertyCh
     val gen = Gen.listOfN(Ed25519.publicKeyLength, Arbitrary.arbByte.arbitrary).map(_.toArray)
 
     forAll(gen) { bytes: Array[Byte] =>
-      val asHex           = Base16.encode(bytes)
+      val asHex = Base16.encode(bytes)
```

Comment:
Having those formatting related changes in virtually every PR is getting annoying